Misc Bug Removal

printer16 opened the pull request:
vampire_powers.dm ~ vamps now know if they finished enthralling #1705
utility.dm ~unconscious people cant feel things injected anymore #606
security.dm ~7 days to play as sec now. Same with detective and warden. might need config option enabled. #1973
exodus-1.dm ~ removed useless circuit board #1991
supplypacks.dm ~ Auxilllary Foensic tools crate spawns with a UV light now.#1936
tree_networking.dm ~ gave the malf AI more room to type things up. Also fixed typo #1914
Made the viral infection outbreak say level 5 all the time so it matches up with sound file.

Lohikar commented on code/game/gamemodes/malfunction/newmalf_ability_trees/tree_networking.dm:
```diff
@@ -92,32 +92,101 @@
+	var/reporter = null
+	var/reporttype = input(usr, "Choose whether to use a template or custom report.", "Create Command Report") in list("Template", "Custom", "Cancel")
+	switch(reporttype)
+		if("Template")
```
Why are AIs permitted to send CCIA templates?

printer16 replied:
Because they are hacking the communications network. They should be allowed to send a message closely resembling a CCIA message.

printer16 replied:
Thinking about it, if this can cause issues it can be removed.

skull132 replied:
I'll ping CCIAA leader and GarnItzal.

skull132 replied:
The stipulation was, let the AI only send messages from CCIAMS. The automated system. As to not have issues of staff complaints against CCIAA for something a malf did.

skull132 commented on code/game/gamemodes/malfunction/newmalf_ability_trees/tree_networking.dm:
```diff
@@ -92,32 +92,101 @@
+			post_comm_message(reporttitle, reportbody)
+
+
+	//for (var/obj/machinery/computer/communications/C in machines)
```
Remove if not used.

skull132 commented on html/changelogs/Printer16-BugFixes.yml:
```diff
@@ -0,0 +1,39 @@
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - bugfix: "Vampires now know when they finished enthralling someone."
+  - bugfix: "Auxilllary Foensic Tools crate now spawns with a UV light."
```
Typos. "Auxiliary" and "forensic"

skull132 commented on html/changelogs/Printer16-BugFixes.yml:
```diff
@@ -0,0 +1,39 @@
+changes: 
+  - bugfix: "Vampires now know when they finished enthralling someone."
+  - bugfix: "Auxilllary Foensic Tools crate now spawns with a UV light."
+  - bugfix: "New players have to wait seven days to play as security now."
```
Remove this.

Your fix doesn't actually change anything, as the age restrictions aren't even used atm. The dynamic age restriction system I wrote should be fixed, last time it was used, it was broken for antags. Which is why it was disabled.

printer16 commented:
Addressed the issues. The ai can either sign the message with any name but it doesn't say CCIA or they can leave it blank to get it to say it's from CCIAMS.

printer16 commented:
Decided to remove the AI's ability to add a fake name because it's pretty easy to just add the CCIA tag onto it. Now they can pick: CCIAMS or nothing as a signature.
